### PR TITLE
feat: added runner delay

### PR DESCRIPTION
**Pull request by therealrinku**
# Description
Added delay time for runner
resolves: #1385 

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [x] **Create an issue and link to the pull request.**

[Screencast from 2024-05-02 14-25-59.webm](https://github.com/usebruno/bruno/assets/76877078/3caf7d83-488b-4090-aa78-c7ae6494a43c)

### CODE REVIEW

**Comment by helloanoop** [comment]
Thank you @therealrinku !

@lohxt1 Can you review this?

File: packages/bruno-app/src/components/RunnerResults/index.jsx [hunk 1]
```diff
@@ -167,11 +182,13 @@ export default function RunnerResults({ collection }) {
...
-                    ) : (
+                    ) : item.responseReceived?.status ? (
                       <span className="text-xs link cursor-pointer" onClick={() => setSelectedItem(item)}>
-                        (<span className="mr-1">{get(item.responseReceived, 'status')}</span>
-                        <span>{get(item.responseReceived, 'statusText')}</span>)
+                        (<span className="mr-1">{item.responseReceived?.status}</span>
+                        <span>{item.responseReceived?.statusText}</span>)
                       </span>
+                    ) : (
+                      <span className="danger text-xs">(request failed)</span>
```
Comment (lohxt1) [inline packages/bruno-app/src/components/RunnerResults/index.jsx]: could you also add the onClick handler for the 'request failed' span

`onClick={() => setSelectedItem(item)}`

File: packages/bruno-app/src/components/RunnerResults/index.jsx [hunk 2]
```diff
@@ -116,6 +117,20 @@ export default function RunnerResults({ collection }) {
           You have <span className="font-medium">{totalRequestsInCollection}</span> requests in this collection.
         </div>
 
+        <div className="mt-6">
+          <label>Delay (in ms)</label>
+          <input
+            type="text"
```
Comment (lohxt1) [inline packages/bruno-app/src/components/RunnerResults/index.jsx]: would you mind changing this to the type 'number'

Comment (therealrinku) [reply packages/bruno-app/src/components/RunnerResults/index.jsx]: sure, done!

**Comment by jroucou** [comment]
hello, 
is it possible to provide a parameter for bruno-cli as well ? could be a nice addition.

**Comment by helloanoop** [comment]
Merged! Thanks @therealrinku !

**Comment by ben-sai** [comment]
This menu option is not available when executing runner from the collection / run  vs Runner top of the screen. 

image

**Comment by feanor777** [comment]
> hello, is it possible to provide a parameter for bruno-cli as well ? could be a nice addition.

also looking forward for this feature!

**Pull request merged**